components/usage/pkg/apiv1/usage-report.go

```diff
@@ -125,12 +125,12 @@ func trimStartStopTime(instances []db.WorkspaceInstanceForUsage, maximumStart, m
 	var updated []db.WorkspaceInstanceForUsage
 
 	for _, instance := range instances {
-		if instance.CreationTime.Time().Before(maximumStart) {
-			instance.CreationTime = db.NewVarcharTime(maximumStart)
+		if instance.StartedTime.Time().Before(maximumStart) {
+			instance.StartedTime = db.NewVarcharTime(maximumStart)
```

Comment:
Unrelated to this PR, but manipulating instances here was surprising, especially since they are handed out through the usage report. I don't have enough context, yet, though. Just wanted to share.

Reply:
I agree, and think it would make more sense to manipulate usage records instead of instances.

Reply:
They are not manipulated, as we create copies here (not working on references/pointers).

But had to read twice as well.. :laughing:  :see_no_evil: